hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/util/CombinedHostsFileReader.java

```diff
@@ -42,7 +42,7 @@
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.classification.InterfaceStability;
 import org.apache.hadoop.hdfs.protocol.DatanodeAdminProperties;
-
+import org.apache.hadoop.util.concurrent.SubjectInheritingThread;
```

Comment:
nit, leave the blank line between this and the other package.

I'm trying to keep that import diff down as its where we so often get needless merge conflict